rkt/list.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2014 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//+build linux
+
+package main
+
+import (
+	"fmt"
+	"os"
+)
+
+var (
+	cmdList = &Command{
+		Name:    "list",
+		Summary: "List containers",
+		Usage:   "",
+		Run:     runList,
+	}
+)
+
+func init() {
+	commands = append(commands, cmdList)
+}
+
+func runList(args []string) (exit int) {
+	n := int(0)
+
+	if err := withContainerHandles(includeContainersDir|includeGarbageDir, func(ch *containerHandle) error {
+		if n == 0 {
```

Comment:
Print this by default and consider adding a `--no-legend` flag if you want to be able to omit it.